crates/core/src/protocols/fri/verify.rs

```diff
@@ -2,6 +2,7 @@
 
 use std::iter;
 
+use binius_fast_compute::fri::{fold_chunk, fold_interleaved_chunk};
```

Comment:
Verifier code should not depend on `binius_fast_compute` methods. These methods should be moved to a common place, either `binius_math` or `binius_ntt`. I recommend making a module called `fri` in `binius_ntt` for now and putting them there. We might clean up / reorganize at a later time.